cura/Settings/MachineManager.py

```diff
@@ -804,6 +805,12 @@ def updateNumberExtrudersEnabled(self):
     def defaultExtruderPosition(self):
         return self._default_extruder_position
 
+    ##  This will fire the propertiesChanged for all settings so they will be updated in the front-end
+    def forceUpdateAllSettings(self):
+        property_names = ["value", "resolve"]
```

Comment:
I'd think you would only need to emit the functions that call the `extruderValue` or `extruderValues` functions. Check for that instead of emitting all 400-something settings to be changed. It may be fast now, but in the future this can potentially slow it down a lot.